allennlp/modules/matrix_attention/__init__.py

```diff
@@ -0,0 +1,6 @@
+
+
```

Comment:
Can you import that base class here, also?

Reply:
Also why?